app/admin/golem/jobs/page.tsx

```
@@ -375,11 +403,56 @@ export default function JobsPage() {
-            <span className="text-blue-400">{newCount} new</span>
+        {/* Quick filter chips */}
+        <div className="flex items-center gap-2 flex-wrap">
+          <button
+            onClick={() => setFilterStatus('all')}
+            className={`px-3 py-1 rounded-full text-xs font-medium transition-colors ${
+              filterStatus === 'all'
+                ? 'bg-white/20 text-white'
+                : 'bg-white/5 text-white/60 hover:bg-white/10'
+            }`}
+          >
+            All ({filteredCount})
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)